common/djangoapps/util/views.py

```diff
@@ -321,16 +335,25 @@ def get_feedback_form_context(request):
 
     context["subject"] = request.POST["subject"]
     context["details"] = request.POST["details"]
-    context["tags"] = dict(
-        [(tag, request.POST[tag]) for tag in ["issue_type", "course_id"] if tag in request.POST]
-    )
+
+    context["tags"] = {}
+    if "issue_type" in request.POST:
+        context["tags"]["issue_type"] = request.POST["issue_type"]
 
     context["additional_info"] = {}
 
     if request.user.is_authenticated():
         context["realname"] = request.user.profile.name
         context["email"] = request.user.email
         context["additional_info"]["username"] = request.user.username
+
+        course_id = request.POST.get("course_id")
+        if course_id:
+            context["additional_info"]["course_id"] = course_id
+            enrollment = CourseEnrollment.get_enrollment(request.user, CourseKey.from_string(course_id))
+            # TODO: Determine what to do if enrollment is missing or inactive.
+            if enrollment:
```

Comment:
What should we do if enrollment is missing? Should we continue processing the request without it or throw an error? Thoughts?